wandb/cli/cli.py

```diff
@@ -939,35 +939,42 @@ def launch_sweep(
         ctx.invoke(login, no_offline=True)
         api = _get_cling_api(reset=True)
 
-    entity = entity or env.get("WANDB_ENTITY") or api.settings("entity")
+    # if not sweep_config XOR resume_id
+    if not (config or resume_id):
+        wandb.termerror("'config' and/or 'resume_id' required")
+        return
+
+    parsed_user_config = sweep_utils.load_launch_sweep_config(config)
```

Comment:
parse config before entity/project setting, to include `project` and `entity` fields set in the sweep config.